buildSrc/src/main/java/Versions.kt

```diff
@@ -6,7 +6,7 @@ object Versions {
     // Compose
     const val COMPOSE_BOM = "2024.02.00"
     const val COMPOSE = "1.6.1"
-    const val COMPOSE_COMPILER = "1.4.3"
```

Comment:
These Versions won't be used any further, please use the version catalog instead.

Reply:
Well they are used only for tests and the sample app (which has some issues), but I can remove them

Reply:
Done with the latest commit!